### ISSUE TITLE
Question about aggregates

### ISSUE THREAD

**Issue by demamoh** [marcusolsson/goddd#22]
Hello Marcus,

New ddd guy here, and I wanted to ask you if the cargo struct is considered to be an aggregate.

If possible labeling of tactical type in the comments would help as a reference example.

**Comment by marcusolsson** [comment]
You're right, `Cargo` is indeed an aggregate. Documenting the type would also be a good idea. I'll try to find the time to update the documentation.

**Comment by demamoh** [comment]
Thank you.
